components/gfx_traits/print_tree.rs

```diff
@@ -22,20 +22,29 @@ impl PrintTree {
         }
     }
 
-    /// Descend one level in the tree with the given title.
-    pub fn new_level(&mut self, title: String) {
+    /// Descend one level in the tree with the given items string.
+    pub fn new_level(&mut self, queued_item: String) {
```

Comment:
I didn't see a queue here?

Reply:
This is the what `queued_item` give. the origin "title" is not suitable.
```
│  │  │  │  │         sc=StackingContextId(0)
│  │  │  │  │         pos=LogicalRect(H LTR, i720px×b36px, @ (i152px,b2636.883333333333px))
│  │  │  │  │         floatspec-in=L 0px R 0px
│  │  │  │  │         floatspec-out=L 0px R 0px 
│  │  │  │  │         overflow=Overflow { scroll: TypedRect(750px×36px at (-15px,0px)), paint: TypedRect(750px×36px at (-15px,0px)) }
│  │  │  │  │         children=1
│  │  │  │  │         damage=BUBBLE_ISIZES
```

Reply:
maybe `queued_title` better